php/class-wp-cli.php

```diff
@@ -210,30 +210,36 @@ public function assign_coauthors( $args, $assoc_args ) {
 	 * @since 3.0
 	 *
 	 * @subcommand assign-user-to-coauthor
-	 * @synopsis --user_login=<user-login> --coauthor=<coauthor>
+	 * @synopsis --user_login=<user-login> --user_id=<user-id> --coauthor=<coauthor>
```

Comment:
I'm not at all sure how to specify that *one* of `user_login` or `user_id` is required. Anyone else?